google-cloud-spanner/src/test/java/com/google/cloud/spanner/TypeTest.java

```diff
@@ -289,34 +288,46 @@ public void emptyStruct() {
   @Test
   public void structFieldIndexNotFound() {
     Type t = Type.struct(StructField.of("f1", Type.int64()));
-
-    expectedException.expect(IllegalArgumentException.class);
-    expectedException.expectMessage("Field not found: f2");
-    t.getFieldIndex("f2");
+    try {
+      t.getFieldIndex("f2");
+      fail("");
```

Comment:
nit: Here and below, please add a text like "Expected exception"

Reply:
Done